studio/src/components/Chat/DevBotFooter.tsx

```diff
@@ -1,14 +1,16 @@
 import '@/styles/components/DevBotFooter.scss';
-import { DefaultButton, IconButton, Stack, TextField, makeStyles } from "@fluentui/react";
+import { DefaultButton, IconButton, Stack, TextField, Text, makeStyles } from "@fluentui/react";
 import React from "react";
-
+import '../../styles/selectedPlugin.css';
 
 interface props {
     disableSend: boolean
     sendMessageToWss: Function
     onFileChange: any
     pluginStoreToggle: Function
     inputText: string
+    selectedPlugins: any[]
```

Comment:
Is there a way to make the data type into a `set`? Then you don't need to worry about checking whether it was added earlier or not

Reply:
Done